tests/steamship_tests/utils/deployables.py

```diff
@@ -150,6 +150,8 @@ def _check_user(client, instance):
 
 def _delete_deployable(instance, version, deployable):
     instance.delete()
+    version.delete()
```

Comment:
From the description, I understand that `deployable.delete()` cascades to versions. 

Would consider adding a test case that only calls `deployable.delete()` without a call to `version.delete()`